root/static/scripts/common/components/Annotation.js

```diff
@@ -19,17 +19,24 @@ import entityHref from '../utility/entityHref';
 import Collapsible from './Collapsible';
 import EditorLink from './EditorLink';
 
+type EntityProp = $ReadOnly<MinimalCoreEntityT & {
+  +latest_annotation?: AnnotationT,
+}>;
+
 type Props = {|
   +$c: CatalystContextT | SanitizedCatalystContextT,
-  annotation: ?{...AnnotationT, editor: SanitizedEditorT},
+  +annotation: ?({+editor: EditorT | SanitizedEditorT | null} & AnnotationT),
```

Comment:
Used twice already and likely to be used for other entity types index, could be its own type.

Reply:
This is local to Annotation.js (doesn't need to appear in any index file). There is also a slight difference in the types in that `editor` is (intentionally) read-only here and not below, though you could make `MaybeSanitizedEditorT` as an alias for `EditorT | SanitizedEditorT | null` since it's also used in EditorLink.js.

Reply:
Oh right, thought it was in some RG file.